### PR TITLE
First Commit for ke-twitter-demo

### PR DESCRIPTION
**Pull request by maurya-anuj**
Adding application and vendor files required to run the demo.

### CODE REVIEW

File: ke-twitter-demo/Pi_app/offline.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	err := cli.Connect(&client.ConnectOptions{
+		Network:  "tcp",
+		Address:  "localhost:1883",
+		ClientID: []byte("send-client"),
+	})
+	if err != nil {
+		panic(err)
+	}
+	optional := false
+	songname := "killbill"
```
Comment (rohitsardesai83) [inline ke-twitter-demo/Pi_app/offline.go]: we need to make the song name optional. we can discuss if we need this .

**Comment by rohitsardesai83** [comment]
@maurya-anuj , please split this into 2 commits , one for the vendored code , and second for the actual code logic.

File: ke-twitter-demo/Pi_app/trackplayer.go [hunk 2]
```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/kubeedge/kubeedge/cloud/pkg/devicecontroller/types"
+	"github.com/yosssi/gmq/mqtt"
+	"github.com/yosssi/gmq/mqtt/client"
```
Comment (sids-b) [inline ke-twitter-demo/Pi_app/trackplayer.go]: seperate golang, vendor and kubeedge packages

**Comment by sids-b** [comment]
@maurya-anuj , please delete the certificates

**Pull request merged**